app/src/main/java/fr/free/nrw/commons/theme/NavigationBaseActivity.java

```diff
@@ -227,6 +232,48 @@ public boolean onNavigationItemSelected(@NonNull final MenuItem item) {
         }
     }
 
+    private String getTechnicalInfo() {
```

Comment:
It seems that you didn't move the method to `Utils` class. Instead of duplicating code it would be nice to have both places using the same method. :)

Reply:
@maskaravivek the getExtraInfo() contains username which needs session manager

Reply:
@maskaravivek I might not have understood correctly, please correct me if I am wrong :)

Reply:
I don't mind including username while sending feedback. 

Make `getExtraInfo` in `CommonsLogSender` `public` and modify it to add any other meta info if you want.

Reply:
@maskaravivek Done! Thank you for a great code review, I appreciate the importance of using common method :)